docs/catalog-description.rst

```diff
@@ -69,21 +69,30 @@ Version 1.8 includes improved star positions and proper motions from Gaia
 DR3, as well as improved estimated magnitudes derived from Gaia DR3 photometry
 and more than 94,000 ACA observed stars. 
 
+
+Use of the AGASC 1.8 in Chandra Operations
+==========================================
+
+The Chandra Aspect Team uses an HDF5 version of the AGASC 1.8 data and
+the Python agasc package https://sot.github.io/agasc/index.html. References
+in this document to the agasc package refer to that Python code.
+Contact aspect_help@cfa.harvard.edu if you would like to use the catalog
+directly with that Python package.
+
+
 Organization of the FITS Data Files
 ===================================
 
 The top directory includes the files::
 
     ----------------------------------------------------------------
-    readme.txt    - Introduction. ASCII
```

Comment:
Question here if we should add catalog-description.rst to the file list or not bother.

Reply:
I was thinking this file would be renamed readme.txt when placing it with the fits file distribution.

The `readme.txt` could be replace by `this file`

Reply:
either way, just removing the line is ok for me.

Reply:
OK, if you were thinking that it was fine to drop the ".rst" extension and just call catalog-description.rst the new readme.txt that's fine too.

Reply:
Done.